### DISCUSSION TITLE
fclones remove  on windows errors out

### DISCUSSION THREAD

**Discussion by ttohumcu** [pkolaczk/fclones#248] [General]
Great tool but i am having issues with remove command.
Using Powershell on windows. Running the following command executes and creates the list of duplicate items on the file specified.
 .\fclones.exe group s:\master\new-photos\1997> 1997.txt
1997.txt contains something like this below.
=====
# Report by fclones 0.32.2
# Timestamp: 2023-10-30 15:44:10.475 -0400
# Command: 'S:\master\new-photos\organize\fclones.exe' group 's:\master\new-photos\1997'
# Base dir: \\\\?\\UNC\\192.168.1.40\\spider\\master\\new-photos\\organize
# Total: 6713049 B (6.7 MB) in 268 files in 109 groups
# Redundant: 3972620 B (4.0 MB) in 159 files
# Missing: 0 B (0 B) in 0 files
98686c7d69d7b97aa23ec62751aad1e8, 96084 B (96.1 KB) * 2:
    \\\\?\\UNC\\192.168.1.40\\spider\\master\\new-photos\\1997\\1997_10\\1997_10_03\\scream_1.jpg
    \\\\?\\UNC\\192.168.1.40\\spider\\master\\new-photos\\1997\\1997_10\\1997_10_03\\scream_2.jpg
3d897f63f72c57347c1ddc0b5ffd74b6, 43393 B (43.4 KB) * 3:
    \\\\?\\UNC\\192.168.1.40\\spider\\master\\new-photos\\1997\\1997_10\\1997_10_04\\byrant.jpg
    \\\\?\\UNC\\192.168.1.40\\spider\\master\\new-photos\\1997\\1997_10\\1997_10_04\\byrant_1.jpg
    \\\\?\\UNC\\192.168.1.40\\spider\\master\\new-photos\\1997\\1997_10\\1997_10_04\\byrant_2.jpg
742835c419eda23415232f4ed48d1c85, 41014 B (41.0 KB) * 2:
    \\\\?\\UNC\\192.168.1.40\\spider\\master\\new-photos\\1997\\1997_10\\1997_10_11\\s15.jpg
    \\\\?\\UNC\\192.168.1.40\\spider\\master\\new-photos\\1997\\1997_10\\1997_10_11\\s15_1.jpg
9fe4b82067a0a7bac1628118591f7a46, 38115 B (38.1 KB) * 3:
    \\\\?\\UNC\\192.168.1.40\\spider\\master\\new-photos\\1997\\1997_10\\1997_10_04\\portj.jpg
    \\\\?\\UNC\\192.168.1.40\\spider\\master\\new-photos\\1997\\1997_10\\1997_10_04\\portj_1.jpg
    \\\\?\\UNC\\192.168.1.40\\spider\\master\\new-photos\\1997\\1997_10\\1997_10_04\\portj_2.jpg
======
Now trying to remove the duplicates. Running following command.
 .\fclones.exe remove <1997.txt
Getting following error message

At line:1 char:22
+ .\fclones.exe remove <1997.txt
+                      ~
The '<' operator is reserved for future use.
    + CategoryInfo          : ParserError: (:) [], ParentContainsErrorRecordException
    + FullyQualifiedErrorId : RedirectionNotSupported

Any ideas?
Thanks**

**Comment by kapitainsky** [comment]
I think the error message you get is self explanatory.... The '<' operator is not supported by your Powershell. You have two options:

1. Switch to the old good cmd where '<' is supported
2. Learn about redirections in your Powershell

**Comment by ttohumcu** [comment]
Thank you. Duh. Now the format error.
S:\master\new-photos\organize>fclones.exe remove <1997.txt
[2023-11-02 09:24:33.367] fclones.exe: error: Input error: Unknown report format. Supported formats are: default, json

**Reply by kapitainsky** [reply]
Maybe start from the beginning and run `fclones group` in cmd as well.

I am not Windows user myself so can't really test but would not be surprise if there is difference between Powershell and cmd behaviour.

**Reply by ttohumcu** [reply]
I think i am just going to switch over to linux version, i was just playing with windows and having more issues. :(

**Comment by kapitainsky** [comment]
BTW 

Here some info re Powershell redirection

https://stackoverflow.com/questions/2148746/the-operator-is-reserved-for-future-use

**Reply by ttohumcu** [reply]
Thanks.
